### ISSUE TITLE
Do we need a legend?

### ISSUE THREAD

**Issue by congdon** [OpenMaine/covid19-foodmap#44]
The icons were confusing to me at first, partly because they are very cluttered in some views.

**Comment by jonjanelle** [comment]
Do you think point clustering might help this?

I'm thinking of something like https://docs.mapbox.com/mapbox-gl-js/example/cluster/

**Comment by congdon** [comment]
Oh, interesting! I don't know!

It's important to me to determine who our (real and imagined) users are as we're refining this tool. Initial users were/are MPA folks, who presumably have some level of education and technology experience. When we get to the layer of people who desperately need access to food pantries, I think we will have a much broader cross-section of Mainers. But I don't really know the characteristics of the imagined users. They have a smart phone, but no computer? Who ARE they?

I did find the enormous clutter of icons daunting when I first put in my zipcode. Another possible solution to that is to set the zoom differently, so that it's more local to me? (And then I can zoom out, or maybe pan... but it's less daunting at that point.)

Also on first use, there are multiple icons, and I couldn't 'read' them. What are you showing me?

**Comment by jonjanelle** [comment]
I'll look into leaflet legend options to see if we can clarify the map symbols. 

@congdon : Are you prompted to zoom to your current location on https://cs21003bffdabdf6204.z13.web.core.windows.net/map.html ?

This feature should be in place, but it depends on whether a user's browser is allowing access to the device location

**Comment by congdon** [comment]
Not sure what you're asking when you say am I prompted, but the 'locate me' doesn't work at that site, either.

Oh wait, I do know what you mean. When I go to the URL. I just sort of automatically always say NO to those things. It did ask me in Chrome, but I said no and when I go to the site again, it won't ask me again. It asked me in Firefox and I said NO, but I went to the page again, I said YES... but it still didn't locate me. It did not ask me in Safari.
